FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -127,26 +127,26 @@ fi
 opts_SetScriptDescription "Runs the FreeSurfer HCP pipline on data processed by prefresurfer"
 
 # Show usage information
-opts_AddMandatory '--subject' 'SubjectID' 'subject' "Subject ID (required).  Used with --path input to create full path to root directory for all outputs generated as path/subject"
+opts_AddMandatory '--session' 'SessionID' 'session' "Session ID (required).  Used with --path input to create full path to root directory for all outputs generated as path/session"
 
-opts_AddOptional '--subjectDIR' 'SubjectDIR' 'subject' 'path to subject directory required, unless --existing-subject is set' "" "--subject-dir"
+opts_AddOptional '--sessionDIR' 'SessionDIR' 'session' 'path to session directory required, unless --existing-session is set' "" "--subject-dir"
```

Comment:
Same, and some phrasing issues.

```suggestion
opts_AddOptional '--sessionDIR' 'SessionDIR' 'session' "path to session directory, required unless --existing-session is set to true" "" '--subject-dir' '--subjectDIR'
```

Reply:
fixed